src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Command that handles user exiting the program
+ */
+public class ByeCommand extends Command {
+    /**
+     * determines that user wants to exit the program
+     * @return boolean: true
+     */
+    @Override
+    public boolean isExit() {
+        return true;
+    }
+
+    /**
+     * on execution, prints goodbye message
```

Comment:
Same goes for this and some other methods in other classes, 
```suggestion
     * On execution, prints goodbye message
```